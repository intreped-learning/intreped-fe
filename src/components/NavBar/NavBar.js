import React, { useState } from 'react';
import './NavBar.scss';

const NavBar = () => {
  const [isOpen, setIsOpen] = useState(false);

  const toggle = () => setIsOpen(!isOpen);

  return (
    <nav className="nav-bar">
      <h1 className="logo">intrep<span>ed</span></h1>
      <form className="search-options">
        <label htmlFor="category-select">Course Topics</label>
        <select id="category-select">
          <option value="engagement">Engagement Strategies</option>
          <option value="classroom">Classroom Management</option>
          <option value="planning">Lesson Planning</option>
          <option value="culture">Culturally Responsive Teaching</option>
          <option value="data">Data-Driven Instruction</option>
        </select>
        <input type="text" placeholder="Search courses"/>
      </form>
    </nav>
  )
}

export default NavBar;