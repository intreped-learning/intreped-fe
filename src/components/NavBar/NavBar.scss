.nav-bar {
  align-items: center;
  display: flex;
  justify-content: space-between;
  padding: 20px;
  position: fixed;
  top: 0;
  width: 100%;
  z-index: 100;
}

.search-options {
  display: flex;
  justify-content: space-evenly;
  align-items: center;
  width: 50%;
}

.label {
  margin: 0;
}