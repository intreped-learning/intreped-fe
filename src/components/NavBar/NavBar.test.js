import React from 'react';
import { shallow } from 'enzyme';
import NavBar from './NavBar';

describe('NavBar', () => {
  it('should match snapshot', () => {
    const getWrapper = () => shallow(
      <NavBar/>
    );
    const wrapper = getWrapper();
    expect(wrapper).toMatchSnapshot();
  });
});