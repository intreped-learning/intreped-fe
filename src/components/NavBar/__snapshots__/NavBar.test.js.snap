// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match snapshot 1`] = `
<nav
  className="nav-bar"
>
  <form
    className="search-options"
  >
    <label
      htmlFor="category-select"
      id="categories"
    >
      Course Categories
    </label>
    <select
      id="category-select"
    >
      <option
        value="engagement"
      >
        Engagement Strategies
      </option>
      <option
        value="classroom"
      >
        Classroom Management
      </option>
      <option
        value="planning"
      >
        Lesson Planning
      </option>
      <option
        value="culture"
      >
        Culturally Responsive Teaching
      </option>
      <option
        value="data"
      >
        Data-Driven Instruction
      </option>
    </select>
    <input
      placeholder="Search courses"
      type="text"
    />
  </form>
</nav>
`;
